### PR TITLE
add the series URL to series lists

### PR DESCRIPTION
**Pull request by jamesgorrie**
Looks the same, just the headers are links.
Just makes it easier for the editors to get to it as well.

### CODE REVIEW

File: server/controllers/async-controllers.js [hunk 1]
```diff
@@ -17,9 +17,11 @@ const getSeriesData = async(ctx) => {
   const items = promoList.items.toJS();
   const image =  items[0].image;
   const color = seriesResponse ? getSeriesColor(id) : series.color;
+  console.log(series);
```
Comment (davidpmccormick) [inline server/controllers/async-controllers.js]: log?

File: server/views/components/numbered-list/numbered-list.njk [hunk 2]
```diff
@@ -2,7 +2,13 @@
 <div {% if data.sliderId %}data-id="{{ data.sliderId }}"{% endif %} class="numbered-list{% for modifier in modifiers %} numbered-list--{{modifier}}{% endfor %} {% for class in data.classes %} {{class}}{% endfor %} numbered-list--{{ model.color }}">
   {% if model.name %}
     <header class="numbered-list__header {{ 'numbered-list__header--has-image' if model.image }}">
-      <h2 class="numbered-list__heading {{ {s:'WB5', m:'WB4', l:'WB3'} | fontClasses if isTransporter else {s:'WB6'} | fontClasses }}">{{ model.name }}</h2>
+      <h2 class="numbered-list__heading {{ {s:'WB5', m:'WB4', l:'WB3'} | fontClasses if isTransporter else {s:'WB6'} | fontClasses }}">
+        {% if model.url %}
+          <a href="/series/{{ model.url }}" class="no-text-decoration">{{ model.name }}</a>
```
Comment (davidpmccormick) [inline server/views/components/numbered-list/numbered-list.njk]: does `plain-link` accomplish this?

Comment (jamesgorrie) [reply server/views/components/numbered-list/numbered-list.njk]: Yip, changed.

**Comment by jamesgorrie** [comment]
@davidpmccormick good spot. Chaars!

File: server/controllers/async-controllers.js (hunk 1 above)
Comment (jamesgorrie) [reply server/controllers/async-controllers.js]: rebased

**Pull request merged**